src/selectors/themeColors.ts

```diff
@@ -3,7 +3,7 @@ import theme from './theme'
 
 const colors = {
   dark: {
-    bg: 'rgba(0, 0, 0, 1)',
+    bg: '#000000',
```

Comment:
@ankitkarna99 Is there a reason `rgba` does not work? If so, you should add a comment explaining the constraint. Otherwise a developer will not know that there are special restrictions on the format of the color.

I believe rgba was required for web in a previous version of em, but that is no longer the case.